analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UnusedPrivateMember.CSharp8.cs

```diff
@@ -82,3 +82,65 @@ private class MyServiceTransient : IMyService
+        private string Field3;              // Compliant, only read by ??
+        private string Field4;              // Compliant, read by ?? and read and written by ??=
+        private string Field5;              // Compliant, read in Property5_1 and read and written by ??= in Property5_2
+        private string Field6_1;            // Compliant, read and written by 1st ??=
+        private string Field6_2;            // Compliant, read by both ??= and written by 2nd ??=
+        private string Field6_3;            // Compliant, read and written by 2nd ??=
+        private string Field7_1;            // Compliant, read and written by ??=
+        private string Field7_2;            // Compliant, read and written by ??
+        private string Field7_3;            // Compliant, read and written by ??
+        private string Field8;              // Compliant, read and written by ??=
+        private string Field9;              // Compliant, read and written by ??=
+        private string Field10;             // Compliant, read and written by ??=
+        private string Field11;             // Compliant, read and written by ??=
+        private string[] Field12;           // Compliant, item read and written by ??=
+        private string Field13_1;           // Compliant, read and written by ??=
+        private string Field13_2;           // Compliant, read and written by +=
+        private string Field14_1;           // Compliant, read and written by ??=
+        private string Field14_2;           // Compliant, written by = and read by ??=
+        private string Field15_1;           // Compliant, read and written by ??=
+        private string Field15_2;           // Compliant, written by = and read by SomeMethod
+
+        public string Property0 => Field0_3;
+        public string Property1 => Field1 ??= "1";
+        public string Property2 => Field2 ??= Field2;
+        public string Property3 => Field3 ?? "1";
+        public string Property4 => Field4 ?? (Field4 ??= "1");
+        public string Property5_1 => Field5;
+        public string Property5_2 => Field5 ??= Field5;
+        public string Property6 => Field6_1 ??= Field6_2 ??= Field6_3;
+        public string Property7 => Field7_1 ??= (Field7_2 ?? Field7_3);
+        public string Property8 => "1" + (Field8 ??= "1");
+        public string Property9 => SomeMethod(Field9 ??= "1");
+        public string Property10 => this[Field10 ??= "1"];
+        public string Property11 => this.Field11 ?? "1";
+        public string Property12 => Field12[0] ??= "1";
+        public string Property13 => Field13_1 ??= (Field13_2 += "1");
+        public string Property14 => Field14_1 ??= (Field14_2 = "1");
+        public string Property15 => Field15_1 ??= SomeMethod(Field15_2 = "1");
+
+        public UsingCoalescingAssignmentOperator(string sPar) { }
+
+        public string this[string sPar] => sPar + "1";
```

Comment:
AFAICT, this is just scaffolding to enable the testcases above, right?

Let's just return null so we don't need to think about what `sPar` and `+"1"` is testring
```suggestion
        public string this[string sPar] => null;
```